foursight_core/checks/ecs_checks.py

```diff
@@ -17,7 +17,7 @@ def ecs_status(connection, **kwargs):
     client = ECSUtils()
     cluster_arns = client.list_ecs_clusters()
     for cluster_arn in cluster_arns:
-        if 'cgap' in cluster_arn:
+        if 'cgap' in cluster_arn or 'fourfront' in cluster_arn:
```

Comment:
Change from Will 2023-02-02 10:30am.

Reply:
So this will use a function from utils, I presume.

Reply:
Yes, have TODO comment there about it, not sure why not appearing above.